pkg/store/bucket.go

```diff
@@ -1229,6 +1230,12 @@ func (s *BucketStore) Series(req *storepb.SeriesRequest, srv storepb.Store_Serie
 		defer s.queryGate.Done()
 	}
 
+	tenant, err := tenancy.GetTenantFromGRPCMetadata(srv.Context())
+	if err != nil {
```

Comment:
How is the tenant value actually used in store gateway? I see it is only used in the logger rn.

Reply:
That's right for now. Splitting up the full implementation in order to not end up with a very large PR. There's an action plan at the end of [the proposal](https://thanos.io/tip/proposals-accepted/202304-query-path-tenancy.md/). This PR corresponds to step one of the query component.